cli/src/semgrep/rule_match.py

```diff
@@ -475,6 +496,8 @@ def is_blocking(self) -> bool:
                 return False
             else:
                 return blocking
+        elif self.validation_state is not None:
+            return self.is_validation_state_blocking
```

Comment:
If we also happen to send an unconditional block somehow (i.e., `dev.semgrep.actions` has `block`), I would expect this to block still. I wouldn't expect us to send both currently, I guess, but I might prefer
```python
else:
    return self.is_validation_state_blocking or blocking
```
here.